exhibition/static/exhibition/css/exhibitors-public-detail.css

```
@@ -282,3 +282,147 @@
+.exhibition-session-list .pretalx-session-info .title {
+    font-size: 16px;
+    font-weight: 500;
+    margin-bottom: 4px;
+}
+
+.exhibition-session-list .pretalx-session-info .speakers {
+    color: rgba(0, 0, 0, 0.54);
+}
+
+.exhibition-session-list .pretalx-session-info .abstract {
+    display: block;
```

Stylelint (17.14.0)

[error] Duplicate property "display" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)